packages/antd/src/templates/BaseInputTemplate/index.tsx

```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import * as React from "react";
```

Comment:
Why is it that you replaced the simple `React` with `* as React`?

Reply:
> Why is it that you replaced the simple `React` with `* as React`? Couldn't you have done `import { FocusEvent, ChangeEvent } from "react"` instead and remove `React` from below?

The namespace is a type thing and I was trying to be safe for the time being. Yes it can be removed but I'll need to test it first